### PR TITLE
MTV-67: Filter must-gather data collection

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MTV-67

Preview: https://deploy-preview-152--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#using-must-gather_forklift

Ready for QE.

### CODE REVIEW

File: documentation/modules/using-must-gather.adoc [hunk 1]
```diff
@@ -19,9 +21,7 @@ You can collect data for a one-hour or a 24-hour period and view the data with t
 .Procedure
 
 . Navigate to the directory where you want to store the `must-gather` data.
-. Run the `oc adm must-gather` command:
-
-* To gather data for the past hour:
+. Run the `oc adm must-gather` command to gather data for the past hour:
```
Comment (aufi) [inline documentation/modules/using-must-gather.adoc:24]: "Past hour" or 24 hours is not (and was not) part of Forklift must-gather, there is no such time constraint - please remove it.

File: documentation/modules/using-must-gather.adoc [hunk 2]
```diff
@@ -30,11 +30,38 @@ $ oc adm must-gather --image={must-gather}
...
+
+* Virtual machine:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather} \
+  -- VM=<vm_id> NS=<namespace> /usr/bin/targeted
+----
+
+. Optional: Run the `oc adm must-gather` command with the `gather_metrics_dump` script to gather data for the past 24 hours:
```
Comment (aufi) [inline documentation/modules/using-must-gather.adoc:59]: Same as above, please remove "the past 24 hours".

File: documentation/modules/using-must-gather.adoc [hunk 3]
```diff
@@ -30,11 +30,38 @@ $ oc adm must-gather --image={must-gather}
...
+$ oc adm must-gather --image={must-gather} \
+  -- PLAN=<migration_plan> /usr/bin/targeted
+----
+
+* Virtual machine:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather} \
+  -- VM=<vm_id> NS=<namespace> /usr/bin/targeted
```
Comment (aufi) [inline documentation/modules/using-must-gather.adoc:56]: Just for better understading what VM ID it is (not an ID from vmware/rhv/openshift/..), the VM ID is the one from the Plan CR

**Review by aufi — CHANGES_REQUESTED** [review]

File: documentation/modules/using-must-gather.adoc (hunk 2 above)
Comment (apinnick) [reply documentation/modules/using-must-gather.adoc:59]: The content in this section is based on the konveyor/forklift-must-gather. The [gather_metrics_dump](https://github.com/konveyor/forklift-must-gather/blob/main/collection-scripts/gather_metrics_dump) script says that it gathers data for 24 hours. Can you confirm that it does not?

File: documentation/modules/using-must-gather.adoc (hunk 1 above)
Comment (apinnick) [reply documentation/modules/using-must-gather.adoc:24]: Is forklift must gather not using the scripts in konveyor? It looks like the [default](https://github.com/konveyor/forklift-must-gather/blob/main/collection-scripts/gather_metrics) is to collect data for 6 hours.

File: documentation/modules/using-must-gather.adoc (hunk 2 above)
Comment (aufi) [reply documentation/modules/using-must-gather.adoc:59]: Ah, sorry, you're right the gather_metrics_dump script is for 24 hours, so it should be kept.   (just the "default" must gather execution doesn't have a time restriction, but also doesn't dump metrics).

File: documentation/modules/using-must-gather.adoc (hunk 1 above)
Comment (aufi) [reply documentation/modules/using-must-gather.adoc:24]: Correct, the metrics capture script is there, but currently it is not executed by default by the main script https://github.com/konveyor/forklift-must-gather/blob/main/collection-scripts/gather

Comment (apinnick) [reply documentation/modules/using-must-gather.adoc:24]: Yes, I see what you mean. So it looks like the default gathers data about CRs and logs but NOT metrics data. Is that correct?

File: documentation/modules/using-must-gather.adoc (hunk 2 above)
Comment (apinnick) [reply documentation/modules/using-must-gather.adoc:59]: Got it. metrics_dump collects metrics data for 24 hours but default gather only collects logs and CRs. Will fix.

File: documentation/modules/using-must-gather.adoc (hunk 1 above)
Comment (aufi) [reply documentation/modules/using-must-gather.adoc:24]: Correct!

File: documentation/modules/using-must-gather.adoc (hunk 2 above)
Comment (aufi) [reply documentation/modules/using-must-gather.adoc:59]: Correct, thank you!

**Review by aufi — APPROVED** [review]

**Pull request merged**